src/redux/rootReducer.js

```diff
@@ -1,8 +1,10 @@
 import { combineReducers } from "redux";
-import userReducer from "./user/user.reducer";
+import {userReducer} from "./user/user.reducer";
+import {loginReducer} from "./user/user.reducer";
 
 const rootReducer = combineReducers({
-  user: userReducer,
+  user : userReducer,
+  loginUser : loginReducer,
```

Comment:
zasto novi reducer za login? to je sve unutar user reducera, koji bi zapravo trebao da se zove auth ili slicno